iceoryx_integrationtest/iceoryx_integrationtest/test_callback_example.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+import unittest
+
+import launch
+from launch_ros.substitutions import ExecutableInPackage
+import launch_testing
+import launch_testing.actions
+from launch_testing.asserts import assertSequentialStdout
+
+import pytest
+
```

Comment:
You could define and add some meta information about the specific integration test like we [did for the SWE.4 tests](https://github.com/eclipse-iceoryx/iceoryx/blob/master/doc/aspice_swe3_4/example/iceoryx_component/test/moduletests/test_example_base_class.cpp#L29-L33).